src/libcollections/vec.rs

```diff
@@ -1301,6 +1306,47 @@ impl<T: Clone> Vec<T> {
     }
 }
 
+impl<T: Default> Vec<T> {
+    /// Resizes the `Vec` in-place so that `len()` is equal to `new_len`.
```

Comment:
nit: we (the docs team) for the time being, don't add `()` to the end of method names in documentation, so this `len()` should be `len`, ideally linking to the associated method. alternatively, you could just say "...so that its length is equal to `new_len`." not a big deal though